src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -315,7 +360,10 @@ const useCreate = () => {
   return useMutation(mutation, {
     update: (cache, { data }) => {
       const query = workspaces
-      const { metacardsByTag } = cache.readQuery({ query })
+      const { metacardsByTag, user } = cache.readQuery({ query })
+      securityAttributes.forEach(securityAttr => {
+        data.createMetacard[securityAttr] = []
+      })
```

Comment:
This seems like logic that graphql should handle

Reply:
I was having issues with the graphql server not returning attributes that were empty. I can try again though